goal_src/jak2/engine/gfx/ocean/ocean-mid.gc

```diff
@@ -7,3 +7,1191 @@
 
 ;; DECOMP BEGINS
 
+(define ocean-mid-block (new 'static 'vu-function :length #x490 :qlength #x248))
+
+(defmethod ocean-mid-setup-constants ocean ((obj ocean) (arg0 ocean-mid-constants))
+  (let ((v1-0 *math-camera*))
+    (set! (-> arg0 hmge-scale quad) (-> v1-0 hmge-scale quad))
+    (set! (-> arg0 inv-hmge-scale quad) (-> v1-0 inv-hmge-scale quad))
+    (set! (-> arg0 hvdf-offset quad) (-> v1-0 hvdf-off quad))
+    (set-vector! (-> arg0 fog) (-> v1-0 pfog0) (-> v1-0 fog-min) (-> v1-0 fog-max) 3072.0)
+    )
+  (set-vector! (-> arg0 constants) -0.25 -0.5 0.0 393216.0)
+  (let ((t9-0 matrix-local->world)
```

Comment:
I think this function changed in one of Vaser's recent PRs. You might need to regenerate this file.

Reply:
It was actually `matrix-world->local` that changed